config/routes.rb

```diff
@@ -24,6 +24,21 @@
         end
       end
 
+      resources :apps, only: [], param: :name do
+        resources :versions, only: [], param: :number, constraints: { number: /[\d\.]+/ } do
+          resources :builds, only: [], param: :timestamp do
+            resources :visitors, only: [] do
+              resource :config, only: :show, controller: 'app_visitor_configs'
```

Comment:
Here's what this looks like as a path (extracted from rake routes):

`/api/v1/apps/:app_name/versions/:version_number/builds/:build_timestamp/visitors/:visitor_id/config`